test/blocks/quiz/mocks/mock-states.js

```diff
@@ -0,0 +1,116 @@
+export const userSelection = [
+  {
+    selectedQuestion: {
+      questions: 'q-category',
+      'max-selections': '3',
+      'min-selections': '1',
+    },
+    selectedCards: {
+      photo: true,
+      video: true,
+    },
+  },
+  {
+    selectedQuestion: {
+      questions: 'q-rather',
+      'max-selections': '1',
+      'min-selections': '1',
+    },
+    selectedCards: { custom: true },
+  },
+  {
+    selectedQuestion: {
+      questions: 'q-photo',
+      'max-selections': '1',
+      'min-selections': '1',
+    },
+    selectedCards: { organize: true },
+  },
+  {
+    selectedQuestion: {
+      questions: 'q-video',
+      'max-selections': '1',
+      'min-selections': '1',
+    },
+    selectedCards: { social: true },
+  },
+  {
+    selectedQuestion: {
+      questions: 'q-customer',
+      'max-selections': '1',
+      'min-selections': '1',
+    },
+    selectedCards: { individual: true },
+  },
+];
+
+export const answers = [
+  ['q-category', ['photo', 'video']],
+  ['q-rather', ['custom']],
+  ['q-photo', ['organize']],
+  ['q-video', ['social']],
+  ['q-customer', ['individual']],
+];
+
+export const resultRules = [
+  {
+    result: '(ai,ai-edu,ai-bus,ai-ind,au-edu,au-bus,au-ind,an-edu,an-bus,an-ind,ae-edu,ae-bus,ae-ind,lr-edu,lr-bus,lr-ind,id,pr-edu,pr-bus,pr-ind,ps-bus,ps-edu,ps-ind,ac,pdf)&(ai,ai-edu,ai-bus,ai-ind,au-edu,au-bus,au-ind,an-edu,an-bus,an-ind,ae-edu,ae-bus,ae-ind,lr-edu,lr-bus,lr-ind,id,pr-edu,pr-bus,pr-ind,ps-bus,ps-edu,ps-ind,ac,pdf)',
+    'umbrella-result': 'cc',
+    url: 'https://uar-integration--milo--adobecom.hlx.page/drafts/colloyd/uar-results-block/uar-results',
```

Comment:
I have concern for all urls, we should use .live not .page for automation, there is no cache for .page, so we should use .live for all urls

Reply:
Changed all urls to either empty/just sample path.